package/autoyast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Jun 25 08:53:44 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- The language, timezone and keyboard sections are applied and
+  removed during the first stage.
```

Comment:
issue missing here, otherwise it looks reasonable for me

Reply:
Have to check if there is already a global bug report to moving all the config to the first stage by default or will create one.

Reply:
Added